src/components/Author.js

```diff
@@ -3,11 +3,11 @@ import PropTypes from 'prop-types'
 import { Link, graphql } from 'gatsby'
 import Div from 'styled-kit/Div'
 
-import Avatar from './Avatar'
+import twitterLogo from 'content/assets/twitter-logo.svg'
+import codepenLogo from 'content/assets/codepen-logo.svg'
+import githubLogo from 'content/assets/github-logo.svg'
```

Comment:
Not sure these belong in `content` since they're not connected to blog posts. Perhaps we should move them to `static`? 🤔I don't know the difference between those two. Yet